portal-frontend/src/app/shared/mobile/naru-residence-mobile-card/naru-residence-mobile-card.component.scss

```diff
@@ -48,3 +49,35 @@ a {
   padding-right: 30px;
   padding-bottom: 10px;
 }
+
+.text-error {
+  color: colors.$grey-dark;
+  font-weight: 400;
+  display: inline;
+  padding-bottom: 0 !important;
+}
+
+.field-error {
+  color: colors.$error-color;
+  font-size: rem(15);
+  font-weight: 700;
+  display: flex;
+  align-items: center;
+
+  .mat-icon {
+    min-width: rem(24);
+  }
+}
+
+.error-card {
+  border-radius: 4px !important;
```

Comment:
Has this always been in pixels? I think it should be in rems via the `rem` function.